dev/TeachingTip/TeachingTip.cpp

```diff
@@ -144,6 +171,13 @@ void TeachingTip::OnPropertyChanged(const winrt::DependencyPropertyChangedEventA
     {
         OnIconSourceChanged();
     }
+    else if (property == s_TitleProperty
+        || property == winrt::AutomationProperties::AutomationIdProperty()
+        || property == winrt::AutomationProperties::NameProperty())
```

Comment:
When does the AutomationProperties attached property come through here?

Reply:
It doesn't

---
In reply to: [267990166](https://github.com/Microsoft/microsoft-ui-xaml/pull/469#discussion_r267990166) [](ancestors = 267990166)